enigma/src/main/resources/lang/zh_cn.json

```
@@ -8,6 +8,7 @@
 	"mapping_format.tiny_file": "Tiny 文件",
 	"mapping_format.srg_file": "SRG 文件",
 	"mapping_format.proguard": "Proguard 文件",
+	"mapping_format.recaf": "Recaf",
```

```suggestion
	"mapping_format.recaf": "Recaf 文件",
```
The others say 文件 too, to keep it consistent?